modules/app.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable*/
+const time = document.querySelector('.date');
+const monthNames = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun',
+  'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec',
+];
+
+const timer = () => {
+  const date = new Date();
+  const month = date.getMonth();
+  const year = date.getFullYear();
+  const day = date.getDate();
+  time.innerHTML = `${monthNames[month]} ${day} ${year}, ${date.toLocaleTimeString()}`;
+}
+
+setInterval(timer, 1000);
+
+export { timer }
```

Comment:
- [ ] Kindly user the [Luxon Library](https://github.com/moment/luxon/blob/master/docs/install.md) to get the  [Current date and time](https://github.com/moment/luxon/blob/master/docs/tour.md#get-the-current-date-and-time). Kindly install the package using npm and import it into your project using ES6 syntax as stated in the project requirement.

Reply:
Thank you for the feedback. I downloaded and paste the luxon files into a date.js file and imported it into the index.js file. I had so many linter errors that I could not resolve. Please which approach would you suggest for me to forestall linter errors?